packages/ffe-form-react/src/BaseRadioButton.js

```diff
@@ -78,13 +86,13 @@ BaseRadioButton.propTypes = {
     /** The name of the radio button */
     name: string.isRequired,
     /** The selected value of the radio button set */
-    selectedValue: oneOfType([bool, string]),
+    selectedValue: oneOfType([number, string]),
```

Comment:
Same here, for a component that really only has two active states, it makes very little sense not to allow the model to have a boolean value